src/main/kotlin/no/fdk/concept_catalog/service/SkosApNoModelService.kt

```diff
@@ -276,6 +286,72 @@ class SkosApNoModelService(
             }
     }
 
+    private fun addBegrepsRelasjonToConcept(conceptBuilder: ConceptBuilder, concept: Begrep) {
+        concept.begrepsRelasjon
+            ?.let {
```

Comment:
Blir det ikke samme om du dropper mellomsteget med let og kjører forEach direkte?
```suggestion
            ?.forEach {
```